recipe/session/sessmodels/models.go

```diff
@@ -97,6 +113,7 @@ type TypeInput struct {
 	Override                 *OverrideStruct
 	ErrorHandlers            *ErrorHandlers
 	Jwt                      *JWTInputConfig
+	GetTokenTransferMethod   func(req *http.Request, forCreateNewSession bool, userContext supertokens.UserContext) string
```

Comment:
instead of returning string, can't you make it return a struct / enum which has one of cookie, header, or any?

Reply:
Done